uber/models.py

```diff
@@ -1338,6 +1345,16 @@ def requested(self, department):
     def assigned_to(self, department):
         return int(department or 0) in self.assigned_depts_ints
 
+    def trusted_in(self, department):
+        return int(department or 0) in self.trusted_depts_ints
+
+    def assigned_and_trusted_in(self, department):
```

Comment:
that section might be able to be simplified now that there isn't a global 'trusted' flag, or just kill assigned_and_trusted_in() and move that check out into this particular email filter